src/features/court/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import {HalfCourt} from './components/half-court';
+import {SubCourt} from './components/sub-court';
```

Comment:
space between `{` `}` is not necessary in your lint ?

Reply:
yes, it's not. But I suggest I should make it consistent.

Reply:
BTW, my team uses no space convention.
But let's use space in this repo.